src/impl.cpp

```diff
@@ -0,0 +1,9 @@
+#include "impl.h"
+#include <cmath>
+
+bool Quadro(int a,int b,int c, double& x1,double& x2)
+{
+	return (x1 = (-b + (sqrt(static_cast<double>(b*b - 4*a*c))))/(2*a),
```

Comment:
isn't it a copy-paste of https://github.com/smay1613/CITest/blob/bfd9fcb5bea25f83c636033459b1b58a971237c1/src/impl.cpp#L12?

Reply:
so, the same as in https://github.com/smay1613/CITest/pull/24/files